packages/mobile/src/components/lineup-tile/LineupTileTopRight.tsx

```diff
@@ -50,6 +50,10 @@ type Props = {
    * The id of the track
    */
   trackId?: number
+  /**
+   * Whether or not the track is a collection
```

Comment:
nit: whether or not the tile is a collection

Reply:
lol bad robot